README.md

```diff
@@ -38,6 +38,8 @@ jsonFile(filename, function (err, file) {
 			baz: true
 		}
 	});
+	
+	file.remove('description'); // remove a specific key-value pair
```

Comment:
perhaps this should show both the callback and promise version, like `get` above it?